### ISSUE TITLE
For multiple repos, the output is not sorted anymore on repo name

### ISSUE THREAD

**Issue by davbeek** [davbeek/gitinspectorgui#37]
For `--no-multi-core`, the output for multiple repos should be sorted on repo order.

**Comment by Alberth289346** [comment]
Why not always?

I'd consider multi-core an implementation detail for faster execution, a user has no need to get changed behavior from it imho.

**Comment by davbeek** [comment]
Certainly, I would try to get sorted output always, but that would mean delaying output info until. That can lead to long output delays in case the first repo takes a long time to process. Currently, we show the processed repo names as soon as they are ready.

**Comment by Alberth289346** [comment]
Split the process in 2 steps. First do an inventory what repos exist, and display that. Then start processing them.

**Comment by davbeek** [comment]
For big repos, that could be useful. 

For small repos, for which output is generated quickly, it would mean first showing a list of all sorted repos and then very quickly after that the (unsorted) list of repos for which output has been generated.

**Comment by davbeek** [comment]
I don't know exactly when this has been fixed, but it works now. Closing.
